Log warnings about Barrel models

The log is populated with warnings from model loading - barrel item blocks are not found. Both the block and the inventory model seem to be working fine, there's no visible error in game.

Snippet of the error:
```
Exception loading model for variant tfc:wood/barrel/ash#inventory for item "tfc:wood/barrel/ash", normal location exception: 
net.minecraftforge.client.model.ModelLoaderRegistry$LoaderException: Exception loading model tfc:item/wood/barrel/ash with loader VanillaLoader.INSTANCE, skipping
    at ...
Caused by: java.io.FileNotFoundException: tfc:models/item/wood/barrel/ash.json

Exception loading model for variant tfc:wood/barrel/ash#inventory for item "tfc:wood/barrel/ash", blockstate location exception: 
net.minecraftforge.client.model.ModelLoaderRegistry$LoaderException: Exception loading model tfc:wood/barrel/ash#inventory with loader VariantLoader.INSTANCE, skipping
    at ...
Caused by: net.minecraft.client.renderer.block.model.ModelBlockDefinition$MissingVariantException

```

--- a/tfc/blockstates/wood/barrel/acacia.json
+++ b/tfc/blockstates/wood/barrel/acacia.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/ash.json
+++ b/tfc/blockstates/wood/barrel/ash.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/aspen.json
+++ b/tfc/blockstates/wood/barrel/aspen.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/birch.json
+++ b/tfc/blockstates/wood/barrel/birch.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/blackwood.json
+++ b/tfc/blockstates/wood/barrel/blackwood.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/chestnut.json
+++ b/tfc/blockstates/wood/barrel/chestnut.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/douglas_fir.json
+++ b/tfc/blockstates/wood/barrel/douglas_fir.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/hickory.json
+++ b/tfc/blockstates/wood/barrel/hickory.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/kapok.json
+++ b/tfc/blockstates/wood/barrel/kapok.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/maple.json
+++ b/tfc/blockstates/wood/barrel/maple.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/oak.json
+++ b/tfc/blockstates/wood/barrel/oak.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/palm.json
+++ b/tfc/blockstates/wood/barrel/palm.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/pine.json
+++ b/tfc/blockstates/wood/barrel/pine.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/rosewood.json
+++ b/tfc/blockstates/wood/barrel/rosewood.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/sequoia.json
+++ b/tfc/blockstates/wood/barrel/sequoia.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/spruce.json
+++ b/tfc/blockstates/wood/barrel/spruce.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/sycamore.json
+++ b/tfc/blockstates/wood/barrel/sycamore.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/white_cedar.json
+++ b/tfc/blockstates/wood/barrel/white_cedar.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

--- a/tfc/blockstates/wood/barrel/willow.json
+++ b/tfc/blockstates/wood/barrel/willow.json
@@ -19,6 +19,9 @@
         "model": "tfc:barrel_sealed"
       },
       "false": {}
-    }
+    },
+    "inventory": [
+      {}
+    ]
   }
 }

